### PR TITLE
Selenium: cover workspace creation from Ubuntu stacks by selenium tests

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR creates selenium tests that cover workspace creation from next Ubuntu stacks:
- .NET
- C++
- Eclipse Che
- Go
- Java Theia (docker)
- Java Theia (openshift)
- Kotlin
- Node
- PHP
- Rails

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/10270, https://github.com/eclipse/che/issues/10050

#### Docs PR

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromJavaMySqlStackTest.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.dashboard.CreateWorkspaceHelper;
+import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
+import org.openqa.selenium.By;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/** @author Aleksandr Shmaraev */
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromJavaMySqlStackTest.java:42]: Is Aleksandr Shmaraev the real author of the test?

Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromJavaMySqlStackTest.java:42]: Yes. I renamed **WorkingWithJavaMySqlStackTest** test to **CreateWorkspaceFromJavaMySqlStackTest**.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromEclipseCheStackTest.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  @Inject private Dashboard dashboard;
+  @Inject private CheTerminal terminal;
+  @Inject private DefaultTestUser defaultTestUser;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private CreateWorkspaceHelper createWorkspaceHelper;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+
+  @BeforeClass
+  public void setUp() {
+
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromEclipseCheStackTest.java]: empty line looks redundant here

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromEclipseCheStackTest.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    ide.switchToIdeAndWaitWorkspaceIsReadyToUse();
+
+    projectExplorer.waitProjectInitialization(CONSOLE_JAVA_PROJECT);
+    projectExplorer.waitProjectInitialization(WEB_JAVA_SPRING_PROJECT);
+  }
+
+  @Test(priority = 1)
+  public void checkConsoleJavaSimpleProjectCommands() {
+    consoles.executeCommandFromProjectExplorer(
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromEclipseCheStackTest.java:87]: It is not enough to check Eclipse Che stack with Java Simple Project. This stack is dedicated to develop something related to Eclipse Che project including Eclipse Che extensions, and Eclipse Che project itself, and there is separate issue for this staff https://github.com/eclipse/che/issues/10107, which becomes partly resolved.

**Review by dmytro-ndp — APPROVED** [review]

**Comment by SkorikSergey** [comment]
ci-build

**Review by artaleks9 — APPROVED** [review]

**Comment by SkorikSergey** [comment]
ci-build

**Pull request merged**